cli/terminal.go

```diff
@@ -118,3 +122,90 @@ func ReadPlanFromTerminal() (int, error) {
 	}
 	return planID, nil
 }
+
+// formats a list of strings to a tidy column representation
+func columns(data []string) (string, error) {
+	width, _, err := cliDimensions()
+	if err != nil {
+		// workaround for tests because terminal size cannot be calculated
```

Comment:
Hmm :thinking: , wouldn't be better to use strategy pattern and inject different strategy based on if we are in the test or not? So for tests we would inject strategy doing `return strings.Join(data, " "), err` and for regular use we would inject `return formatTable(data, width)`. 
Then the strategy could be also tested separately. What do you think?